<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6912" started-at="2015-05-08T22:51:02Z" finished-at="2015-05-08T22:51:09Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6912" started-at="2015-05-08T22:51:02Z" finished-at="2015-05-08T22:51:09Z">
      <class name="com.ecofactorqa.stateapi.StateAPI_Set_Valid_HVAC_mode">
        <test-method status="PASS" signature="init()[pri:0, instance:com.ecofactorqa.stateapi.StateAPI_Set_Valid_HVAC_mode@90580fa]" name="init" is-config="true" duration-ms="211" started-at="2015-05-08T15:51:02Z" finished-at="2015-05-08T15:51:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="set_thermostat_to_cool_mode()[pri:0, instance:com.ecofactorqa.stateapi.StateAPI_Set_Valid_HVAC_mode@90580fa]" name="set_thermostat_to_cool_mode" duration-ms="6691" started-at="2015-05-08T15:51:03Z" finished-at="2015-05-08T15:51:09Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected status 200. Actual status is :500 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected status 200. Actual status is :500 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ecofactorqa.stateapi.StateAPI_Set_Valid_HVAC_mode.set_thermostat_to_cool_mode(StateAPI_Set_Valid_HVAC_mode.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- set_thermostat_to_cool_mode -->
      </class> <!-- com.ecofactorqa.stateapi.StateAPI_Set_Valid_HVAC_mode -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
